### PR TITLE
fix: outline tree scroll bar

### PR DESCRIPTION
**Pull request by vinayhospete**
#2245

### CODE REVIEW

File: packages/control-property-editor/src/panels/outline/Tree.tsx [hunk 1]
```diff
@@ -75,7 +75,7 @@ export const Tree = (): ReactElement => {
                 // make sure that tree is fully rendered
                 const rect = node.getBoundingClientRect();
                 const outlineContainer = document.getElementsByClassName('section--scrollable')[0];
-                if (rect.top <= 20 || rect.bottom >= outlineContainer?.clientHeight) {
+                if (rect.top <= 75 || rect.bottom >= outlineContainer?.clientHeight) {
```
Comment (marufrasully) [inline packages/control-property-editor/src/panels/outline/Tree.tsx:78]: can you please define variable for  `75` to give some semantic for `75`.

Comment (DanyalAslam) [reply packages/control-property-editor/src/panels/outline/Tree.tsx:78]: Yes sure

**Review by marufrasully — APPROVED** [review]
Thanks @vinayhospete and @DanyalAslam 
Simple straightforward changes. Did not test it locally.

**Review by voicis — APPROVED** [review]
- Straight forward change
- Changeset present
- Tested locally

**Pull request merged**